user/handle.go

```diff
@@ -319,3 +319,54 @@ func LoginWithEmailAndPassword(email, password string) (result model.User, err e
 	result = getResponse(ctx, user)
 	return
 }
+
+// HasPermission ...
+func HasPermission(userID, permission string) (result bool) {
```

Comment:
func này muốn nhanh thì làm thêm in-memory cache (https://github.com/allegro/bigcache) như sau:
- init thì call func A query hết role và permission ra lưu vào mem
- khi có thay đổi (CRUD) thì call lại func A
- khi cần check thì chỉ cần lấy từ mem lên để check

Reply:
done